src/App.js

```diff
@@ -1,25 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
+import Form from './components/Form';
+import {useEffect} from 'react';
+
+
+
+// Info 
+function App() {  
+  
+  useEffect(() => {
+    document.title = "Hotdoggin around";
```

Comment:
If you;re doing this because you want to have a useEffect in your code ignore this:

You can change the title in the index file at public/index.html